libbeat/cmd/instance/beat.go

```diff
@@ -280,7 +275,7 @@ func NewBeat(name, indexPrefix, v string, elasticLicensed bool, initFuncs []func
 			ID:              id,
 			FirstStart:      time.Now(),
 			StartTime:       time.Now(),
-			EphemeralID:     eid,
+			EphemeralID:     metricreport.EphemeralID(),
```

Comment:
on the left `metricreport.EphemeralID()` is essentially checked if it is a valid uuid with `uuid.FromString`, however, this is removed now. Any reason that this check became redundant?

Reply:
ok cool to know I was just double-checking that this was intentional and not something missed 🙂